QUEST_LV_0300.txt

```diff
@@ -1,42 +1,42 @@
-﻿QUEST_LV_0300_20150323_000001
-QUEST_LV_0300_20150323_000002
-QUEST_LV_0300_20150323_000003
-QUEST_LV_0300_20150323_000004
-QUEST_LV_0300_20150323_000005
-QUEST_LV_0300_20150323_000006
-QUEST_LV_0300_20150323_000007
-QUEST_LV_0300_20150323_000008
-QUEST_LV_0300_20150323_000009
-QUEST_LV_0300_20150323_000010
-QUEST_LV_0300_20150323_000011
-QUEST_LV_0300_20150717_000012
-QUEST_LV_0300_20150717_000013
-QUEST_LV_0300_20150717_000014
-QUEST_LV_0300_20150717_000015
-QUEST_LV_0300_20150717_000016
-QUEST_LV_0300_20150717_000017
-QUEST_LV_0300_20150717_000018
-QUEST_LV_0300_20150717_000019
-QUEST_LV_0300_20150717_000020
-QUEST_LV_0300_20150717_000021
-QUEST_LV_0300_20150717_000022
-QUEST_LV_0300_20150717_000023
-QUEST_LV_0300_20150717_000024
-QUEST_LV_0300_20150717_000025
-QUEST_LV_0300_20150717_000026
-QUEST_LV_0300_20150717_000027
-QUEST_LV_0300_20150717_000028
-QUEST_LV_0300_20150717_000029
-QUEST_LV_0300_20150717_000030
-QUEST_LV_0300_20150717_000031
-QUEST_LV_0300_20150717_000032
-QUEST_LV_0300_20150717_000033
-QUEST_LV_0300_20150717_000034
-QUEST_LV_0300_20150717_000035
-QUEST_LV_0300_20150717_000036
-QUEST_LV_0300_20150717_000037
-QUEST_LV_0300_20150717_000038
-QUEST_LV_0300_20150717_000039
+QUEST_LV_0300_20150323_000001	$To support the Knights of Kaliss, we were deployed here from headquarters.{nl} If you want to know more, head over to our headquarters at Verkti Square.
+QUEST_LV_0300_20150323_000002	$I've been a merchant all my life. When Medzio Diena hit, I thought of setting up shop here as I heard it was safest here.
+QUEST_LV_0300_20150323_000003	$I met many wandering Revelators around here. I really think they were...different, and it wasn't just that they were strong.
+QUEST_LV_0300_20150323_000004	$I don't mean to burden the Goddess' priests even more. I just think blindly believing and waiting is a little too much for us normal folk.
+QUEST_LV_0300_20150323_000005	$The seals of this village are closed right now, so something something might've tampered with it? I think the priest might just be paranoid about it all.
+QUEST_LV_0300_20150323_000006	$I hope the priests don't prioritize the seal above everything else here. It sometimes seems like they'd even sacrifice themselves for it.
+QUEST_LV_0300_20150323_000007	$The villagers could feel all sorts of things if the priest was more transparent about everything.{nl} Maybe that's why as much as we have to endure...
+QUEST_LV_0300_20150323_000008	$The evil energy is making this forest decay into a lifeless place.{nl}It is easily the biggest problem this village faces as of now.
+QUEST_LV_0300_20150323_000009	$Goddess...{nl}Please tell me the seal of the village is still okay...
+QUEST_LV_0300_20150323_000010	$From now there will surely still be demons trying to attack and bring down the seal.{nl}I wonder how long will it be before Goddess Austeja can return...
```

Comment:
I guess I can edit them on my PR if you like. I edited Line #1 based on your suggestion in my new PR.